.gitignore

```diff
@@ -4,8 +4,10 @@
 *.map
 coverage/
 node_modules/
+pnpm-lock.yaml
```

Comment:
Same with these: +1 to including this in a separate issue and/or PR?

Guessing you're a happy pnpm user? 😄

Reply:
yes, shall we put this and array-type settings separately?